test/integration/homepage_popular_links_test.rb

```diff
@@ -78,6 +78,7 @@ class HomepagePopularLinksTest < JavascriptIntegrationTest
 
   context "#edit" do
     setup do
+      UpdateService.stubs(:call)
```

Comment:
Do we need this stubbing? Commenting it out doesn't seem to cause any ill-effect.